src/Core/Form/IdentifiableObject/CommandBuilder/Product/StockCommandsBuilder.php

```diff
@@ -48,11 +49,9 @@ public function buildCommands(ProductId $productId, array $formData): array
         $quantityData = $formData['stock'];
         $command = new UpdateProductStockInformationCommand($productId->getValue());
 
-        /*
-         * Quantity field is a @see DeltaQuantityType type so on submits it contains the delta quantity
-         */
-        if (isset($quantityData['quantities']['quantity'])) {
-            $command->setDeltaQuantity((int) $quantityData['quantities']['quantity']);
+        /* @see DeltaQuantityType */
+        if (isset($quantityData['quantities']['delta_quantity']['delta'])) {
```

Comment:
Something that seems weird is that in the FO the input would be `$quantityData['quantities']['delta_quantity']['delta']['value']` but in the handler we can use `$quantityData['quantities']['delta_quantity']['delta']`

I guess it is thanks/because of the transformer in `SubmittableDeltaQuantityType`